assets/css/main.css

```
@@ -0,0 +1,242 @@
+body,
+ul,
+nav {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  margin: 0 auto;
+  height: 100vh;
```

Maybe you want to set the **minimum height instead**.
```suggestion
  min-height: 100vh;
```